controllers/images.js

```diff
@@ -0,0 +1,79 @@
+import Image from "../models/images.js";
+import multer from "multer";
+import fs from "fs";
+
+const storage = multer.diskStorage({
+    destination: (_req, _file, cb) => {
+        cb(null, __dirname + "/imgs");
+    },
+    filename: (_req, file, cb) => {
+        //console.log(file.originalname);
+        cb(null, file.originalname);
+    }
+});
+
+const img = multer({ storage: storage });
+
+const imageController = {
+    async addImages(req, res, _next) {
```

Comment:
Okay so we need to now modify these functions, we don't want to have a `(req, res, _next)` in any of our functions because we want them to be reusable. So for instance in our `addImage` function, it should take only an `image` parameter. See the example below: 

`async getImage(id) {
    try{
      const image  = await Image.findOne({_id: id})
       return image
     } catch (error) {
      return error
      }`